pkg/credentials/credentials.go

```diff
@@ -33,7 +33,6 @@ var (
 			IsCloudProvider:    true,
 			SecretName:         "cloud-credentials",
 			MountPath:          "/credentials",
-			BslSecretName:      "bsl-cloud-credentials-aws",
```

Comment:
Kinda followed the logic we had for the previous ones as I included this for the case of CI, now that we are pushing this for new feature, we expect the user to have created the secret name with key given in the `credentialsFile` right ?

Reply:
I think user should be able to pick which secret is used. My change didn't break functionality so I think it's okay.

Reply:
Prior to this PR, user was expected to have created a secret anyway but the name was hardcoded. This meant I couldn't use a different secret for the carriage return case unless I overwrite the current secret with the fixed name.

This only affected the credentialFile config case which most customers won't be using. With this PR the secret name for registry is now flexible if they specified bsl secret with credentialsFile config.

Reply:
>  we expect the user to have created the secret name with key given in the credentialsFile right?

Yes.